### DISCUSSION TITLE
CUE announcements

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#2022] [Announcements]
GitHub Discussions provides an "Announcements" category. But critically, there is no way to subscribe to _only_ "Announcements" discussions.  

We therefore propose trialling an "announce" discussion (this discussion) that people can subscribe to. 

CUE project maintainers will post new messages to this locked discussion to:

* Announce new Community Calls.
* Announce new releases.
* Announce the creation of new release milestones.
* Announce new proposals.
* Share links to issues of significant interest to the wider CUE Community.
* Share updates on new docs and content on cuelang.org.
* …

This will act as a stream of small announcements that link to detailed information.

Please subscribe to this discussion using the "Subscribe" button on the right hand side, or via the ⋮ (kebab) menu on mobile.

For questions/concerns/other arising from new announcements added as comments to this discussion, please create a new issue/discussion.

**Comment by myitcv** [comment]
We are excited to confirm Tuesday 25th October at 1530 UTC for CUE Community Call.

For more details see and subscribe to https://github.com/cue-lang/cue/discussions/2033

**Comment by Lyoness** [comment]
Join in next **Tuesday, 8 November at 1630 UTC** for our Documentation and Content Call.  This will discuss plans for beta.cuelang.org, and docs and content for the wider CUE ecosystem! 

See details and add your discussion topics to #2055. 

Don't forget to subscribe.

**Comment by myitcv** [comment]
We are excited to confirm Tuesday 29th November at 1630 UTC for our next CUE Community Call!

For more details see and subscribe to #2152

**Comment by myitcv** [comment]
We have just released [v0.5.0-beta.2](https://github.com/cue-lang/cue/releases/tag/v0.5.0-beta.2). Thanks to all those who have helped to test out [v0.5.0-beta.1](https://github.com/cue-lang/cue/releases/tag/v0.5.0-beta.1) and [v0.5.0-alpha.1](https://github.com/cue-lang/cue/releases/tag/v0.5.0-alpha.1). For now the release notes are just the commit log. We will draft proper release notes for the final v0.5.0 release.

https://github.com/cue-lang/cue/releases/tag/v0.5.0-beta.2

**Comment by Lyoness** [comment]
### Planned Docs and Content call postponed.  

In our last [CUE Community Call 2023-01-31](https://github.com/cue-lang/cue/discussions/2219), we planned a docs-and-content call for Tuesday, February 14th. This call has been postponed, with the new date TBD.

**Comment by myitcv** [comment]
We are excited to confirm Wednesday 12 April, 2023 at 1430 UTC for our next CUE Community Call.

Lots to cover, including a revised version of the modules proposal which will be shared before the call!

For more details see and subscribe to https://github.com/cue-lang/cue/discussions/2326

**Comment by Lyoness** [comment]
CUE is headed to Amsterdam for [Kubecon EU 2023](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/) next week. 

Dates: Tuesday-Friday 18 April

For more details and to schedule some time with the team, please see https://github.com/cue-lang/cue/discussions/2340

**Comment by myitcv** [comment]
We have just published v2 of a proposal for package management in CUE.

The proposal is linked from https://github.com/cue-lang/cue/discussions/2330, where you can also ask questions and provide feedback.

**Comment by myitcv** [comment]
We have just released CUE v0.5.0. This release focuses on changes to the implementation of comprehensions, the algorithm for detecting cycles, and the reimplementation of let comprehensions as fields.

Thanks to all of those who helped to test out all the pre releases and raise bugs.

See https://github.com/cue-lang/cue/releases/tag/v0.5.0 for more details and full release notes.

**Comment by myitcv** [comment]
We have just released CUE v0.6.0-alpha.1. This release focuses on the introduction of required fields.

See https://github.com/cue-lang/cue/releases/tag/v0.6.0-alpha.1 for more details, including a link to the required fields proposal implemented in this release.

**Comment by myitcv** [comment]
On the [2023-04-12 Community Call](https://github.com/cue-lang/cue/discussions/2326) we discussed the plan for upcoming releases of CUE. We have just posted https://github.com/cue-lang/cue/discussions/2335 with more detail, and answers to some frequently asked questions. Please feel free to comment with questions, comments and feedback.

**Comment by myitcv** [comment]
We have just released CUE v0.6.0-alpha.2. As announced on our recent community call, this release includes preliminary support for Wasm.

See https://github.com/cue-lang/cue/releases/tag/v0.6.0-alpha.2 for more details.
